worlds/ladx/LADXR/itempool.py

```diff
@@ -68,11 +68,12 @@
 
 
 class ItemPool:
-    def __init__(self, logic, settings, rnd):
+    def __init__(self, logic, settings, rnd, stabilize_item_pool):

```

Comment:
```suggestion
    def __init__(self, logic, settings, rnd, stabilize_item_pool: bool):
```
It's good practice to add typing when you add new stuff. Makes it easier for future changes.
If you know the typing the other parts of this should have, I'd recommend adding those too (unless they lead to errors, then maybe don't do it, or do it in a separate PR)

Reply:
Alternatively, it's using the `world` object to get `logic`, `settings`, `rnd`, and `stabilize_item_pool`. So it may be preferable to just load the world class in instead of the individual items here.

Reply:
It's small but I'd prefer to touch the upstream LADXR code as little as possible